packages/core/src/modules/dids/domain/keyDidDocument.ts

```diff
@@ -27,7 +33,7 @@ const didDocumentKeyTypeMapping: Record<KeyType, (did: string, key: Key) => DidD
   [KeyType.P256]: getJsonWebKey2020DidDocument,
   [KeyType.P384]: getJsonWebKey2020DidDocument,
   [KeyType.P521]: getJsonWebKey2020DidDocument,
-  [KeyType.K256]: getJsonWebKey2020DidDocument,
+  [KeyType.K256]: getSecp256k1DidDoc,
```

Comment:
Let's use JsonWebKey by default. This seems to be one of the two defaults in DataIntegrity. I think they're moving away from key specific types

Reply:
- Updated to `JsonWebKey` as default
- Should we later add a way to use different didDoc If someone specifically wants to use them ?

Reply:
Yeah that might be useful -- currently they can call the `getSecpVerificationMethod` and construct a did doc on their own.